### PR TITLE
Begin Molly's blogpost on symbulation project

### PR DESCRIPTION
**Pull request by mkmmnn**

### CODE REVIEW

**Comment by mmore500** [comment]
via @anyaevostinar The goal is to develop a web browser GUI of an artificial life system with symbionts that can be incorporated into Avida-ED 5. The prototype will use Symbulation in its current form, but it will aim to match Avida-ED’s GUI so that it can be easily incorporated into Avida-ED 5 when it is ready. The challenges are determining intuitive ways to portray symbionts in an Avida-ED style and implementing that design.

**Comment by djrain** [comment]
Rough draft looks pretty good!

There are some typos, but otherwise it was a pleasure to read. I like how you talked about difficulties and future directions.


My main suggestions are:
- In the intro, you could give a bit more context on what symbulation is, and what are hosts/symbionts
- Split long paragraphs into subsections or just smaller paragraphs
- Add some images
   - e.g. a screenshot of the app, the annoying error squiggles, hover-and-show issues, etc.
- Maybe a few gifs and emoji just for fun
- Fix typos

Here are the typos I noticed:
color-blind-firendly
default random see
Uses can toggle
uses must achieve success on the current challenge by chaning
as long as the mouse movse on the canvas
Suggetions
has been used for research surrounding symbionts (researching?)
Changing the SymOrg in this was
similar to the way tht
Whenever we came accorss
for a few days in a for
self-displine

Looking forward to seeing how it turns out!

**Comment by kayakingCellist** [comment]
Your project sounds really interesting, especially the game aspect!

As someone with not a lot of background in symbiosis and the different ways organisms interact, I was a little unsure of some terms used. Maybe you could briefly explain what hosts and symbionts are as well as vertical transmission and spatial structure. 

The features sound neat. Screenshots of these features in the app could also help you show them off.

I like how you provide steps for getting the program running locally.

Are the difficulties mentioned things to look out for when others go to add to the code (not the end user)? If so, maybe make that more clear so readers don't think the app is not ready to use.

I like all the points you make in the Future Directions and Conclusion and Takeaways sections. (I'm happy to see a future application for the config panel too!) It may be easier to follow if there were several smaller paragraphs or bullets rather than one large chunk.

Overall, I think it's coming along nicely!

File: blog/_posts/2020-06-03-molly-symbulation-blogpost.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
... grid, the project also features input fields that are implemented so that they have proper error-checking capabilities that allow the user to set **vertical transmission** and **spatial structure** for the population. Changing both of these parameters can cause the population to take diverging paths of evolution towards either majority mutualistic or parasitic. The project also boasts buttons that instantly control the animation. The pause/start button toggles the animation's activeness and a reset button allows the user to begin the simulation from the original default random see. Finally, the project has a learning game added. Uses can toggle the game mode on and off. While the game mode is on, the user is prompted with assignments. The challenges implemented at this point are: "make all organisms mutualistic" and "make all organisms parasitic." Before progressing to the subsequent challenge, uses must achieve success on the current challenge by chaning the **vertical transmission** and **spatial structure** settings. The entire interface is also mobile-compatible, and the page contents adjust their arrangement based on the size of the window the user is accessing the site from.
```
Comment (kayakingCellist) [inline blog/_posts/2020-06-03-molly-symbulation-blogpost.md:13]: Can you clarify what "50 times 50 host" means? 50 by 50 grid of hosts?

File: blog/_posts/2020-06-03-molly-symbulation-blogpost.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+4.  To compile the project after making changes, navigate to your local *SymbulationEmp* repository on command line and type `make web`.
+5.  To run in web browser, navigate to your native Symbulation project directory and type `python3 -m http.server` in your command line. It is recommended that you use a separate terminal specifically for serving your local project.
+6.  Navigate to *http://localhost:8000/web/* to see the web interface. Typically using Chrome as the browser presents no issues. 
+
+# Difficulties
+#### Issues with hover-and-show
+- Context: the virtual petri dish uses colored squares and dots to represent organisms. However, the information provided by colors is still limited. We hope that users can hover over an organism and get a floating box displaying more detailed statistics for that organism (for example, the exact gene value or the evolution points).
+- Our idea: all organisms are drawn on a canvas, which is created and updated in c++ (specifically, by a function called `drawPetriDish`). So we decided to:
```
Comment (kayakingCellist) [inline blog/_posts/2020-06-03-molly-symbulation-blogpost.md:29]: probably try be consistent with c++ vs C++

**Comment by szendejo** [comment]
Overall, I think this is an excellent post. Each section is laid out very well and explained in a clear and concise manner. As someone with a limited evolutionary background, it was very easy to follow. It is really neat to go to the GUI and see the Symbulation GUI in action. Nice job!

Here are my suggestions:
- "The most important part of this project was the colorful virtual petri dish (50 times 50 hosts)" I would suggest adding additional background information for what 50 times 50 hosts mean.
- "color-blind-firendly" Typo on friendly
- "Uses can toggle the game mode on and off" Typo on Users
- "Before progressing to the subsequent challenge, uses must achieve" Typo on users 
- "Our idea: all organisms are drawn on a canvas, which is created and updated in c++ (specifically, by a function called drawPetriDish). So we decided to: " For the list, I would suggest turning it into a numbered list since the information presented looks like it is sequential.
- "However, one thing to caution is that the html objects" I would suggest changing html to HTML for consistency.
- For the Future Directions section, I would suggest breaking it up a bit, maybe doing some paragraph breaks for each possible direction. 
- "Whenever we came accorss unfamiliar functions" Typo on across 
- "It is often easy to get stuck on working on a particular project for a few days in a for." I would suggest adding clarification to the sentence.

**Pull request merged**